packages/auth/src/graphql/schema.graphql

```diff
@@ -0,0 +1,71 @@
+type Query {
+  "Fetch a page of grants."
+  grants(
+    "Paginating forwards: the cursor before the the requested page."
+    after: String
+    "Paginating backwards: the cursor after the the requested page."
+    before: String
+    "Paginating forwards: The first **n** elements from the page."
+    first: Int
+    "Paginating backwards: The last **n** elements from the page."
+    last: Int
+  ): GrantsConnection!
+}
+
+type Mutation {
+  "Revoke Grant"
+  revokeGrant(input: RevokeGrantInput!): RevokeGrantMutationResponse!
+}
+
+type PageInfo {
+  "Paginating forwards: the cursor to continue."
+  endCursor: String
+  "Paginating forwards: Are there more pages?"
+  hasNextPage: Boolean!
+  "Paginating backwards: Are there more pages?"
+  hasPreviousPage: Boolean!
+  "Paginating backwards: the cursor to continue."
+  startCursor: String
+}
+
+type GrantsConnection {
+  pageInfo: PageInfo!
+  edges: [GrantEdge!]!
+}
+
+type GrantEdge {
+  node: Grant!
+  cursor: String!
+}
+
+input RevokeGrantInput {
+  continueId: String!
+  continueToken: String!
+  # identifier: String!
+  # client: String!
+  # access: Access!
```

Comment:
```suggestion
  id: String!
```
I _think_ we should be able to delete by grant id in the admin api.

`grantService.deleteGrant` can also take `id` instead of `continueId`, since everywhere we're calling it has the `id` (or the full grant with its `id`)
https://github.com/interledger/rafiki/blob/e518e2c9960da4cbe69328b9281c152f2d758101/packages/auth/src/grant/service.ts#L32